src/DependencyInjection/Compiler/GlobalVariablesCompilerPass.php

```diff
@@ -27,6 +27,8 @@ class GlobalVariablesCompilerPass implements CompilerPassInterface
     public function process(ContainerBuilder $container)
     {
         $container->getDefinition('twig')
+            ->addMethodCall('addGlobal', ['sonata_configuration', new Reference('sonata.admin.configuration')])
```

Comment:
No blocker, more a personal choice. I would prefer `sonata_config`

Reply:
I like it, changed.